### PR TITLE
Adding a root README.md for adding wiki text to home.

### PR DESCRIPTION
**Pull request by branflake2267**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Contributing
+Fork, modify and create a pull create against the master.
```
Comment (branflake2267) [inline CONTRIBUTING.md]: fixing

**Comment by niloc132** [comment]
as in gwt-driver, post full license, rather than a reference to it? isn't that standard, or am i thinking of what gpl requires?

**Comment by branflake2267** [comment]
Its posted now. I added what apached told me to do on the site.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+GXT and GWT bindings for Selenium WebDriver.
+
+##About
+* <a href="https://github.com/niloc132/gxt-driver/wiki" target="_blank">Concepts</a>
+
+##Documentation
+* <a href="https://github.com/niloc132/gxt-driver/wiki">Wiki Home</a>
+
+##Download
+* [[Maven Configuration]]
```
Comment (niloc132) [inline README.md]: probably also formatted funny, waiting for changes from gwt-driver to be fixed

**Pull request merged**